packages/user-account/src/useCases/user.service.js

```diff
@@ -18,7 +18,16 @@ function validateUser(model) {
   return data;
 }
 
+function updateUser(model) {
+  const data = validatorHandler(updateUserSchema, model);
+  if (data !== model) {
+    return boom.badData(data);
+  }
+  return data;
```

Comment:
what do you think use ternary operator for this case?

````js
return data !== model ? boom.badData(data) : data;
````

Reply:
in this case in te future we will add the logic to save the data in sequelize(postgres)